config/routes.rb

```diff
@@ -10,10 +10,10 @@
     resources :sections, only: [:update]
     resources :lessons, only: [:update]
     resources :sections, only: [] do
-      resources :lessons, only: [:create]
+      resources :lessons, only: [:create, :new]
```

Comment:
The links you have on the `instructor/courses#show` page link out to the new forms.  Lessons 35 and 36 will change this, but for now we need these to be present.